### PR TITLE
Delete filename from pending table if actual file doesn't exist just before processing

### PR DESCRIPTION
**Pull request by kuldeepk3**
Delete filename from pending table if actual file doesn't exist just before processing

### CODE REVIEW

File: pravega-sensor-collector/src/main/java/io/pravega/sensor/collector/util/TransactionStateDB.java [hunk 1]
```diff
@@ -40,6 +40,14 @@ public interface TransactionStateDB {
      */
     public void addCompletedFileRecord(String fileName, long beginOffset, long endOffset, long newNextSequenceNumber, Optional<UUID> txnId) throws SQLException;
 
+    /**
+     * Delete record from endingFiles table
```
Comment (ShwethaSNayak) [inline pravega-sensor-collector/src/main/java/io/pravega/sensor/collector/util/TransactionStateDB.java:44]: typo "pendingFiles table"

File: pravega-sensor-collector/src/main/java/io/pravega/sensor/collector/file/FileProcessor.java [hunk 2]
```diff
@@ -164,6 +166,12 @@ void processFile(FileNameWithOffset fileNameWithBeginOffset, long firstSequenceN
             writer.abort();
         }
 
+        File pendingFile = new File(fileNameWithBeginOffset.fileName);
+        if(pendingFile.exists()){
```
Comment (ShwethaSNayak) [inline pravega-sensor-collector/src/main/java/io/pravega/sensor/collector/file/FileProcessor.java:170]: If file does not exist then log the warning message and delete from pending file list

Comment (kuldeepk3) [reply pravega-sensor-collector/src/main/java/io/pravega/sensor/collector/file/FileProcessor.java]: updated.

**Review by ShwethaSNayak — APPROVED** [review]
LGTM

**Review by sachin-j-joshi — APPROVED** [review]
LGTM

**Pull request merged**